validators/iTunes-validator.ts

```diff
@@ -7,39 +7,63 @@ import hostInfo = require("../host-info");
 
 export class ITunesValidator implements Mobile.IiTunesValidator {
 	private static NOT_INSTALLED_iTUNES_ERROR_MESSAGE = "iTunes is not installed. Install it on your system and run this command again.";
+	private static BITNESS_MISMATCH_ERROR_MESSAGE = "The bitness of Node.js and iTunes must match. Verify that both Node.js and iTunes are 32-bit or 64-bit and try again.";
```

Comment:
WE can use: Verify that both Node.js and iTunes are respectively 32-bit or 64-bit and try again."